setup.py

```diff
@@ -75,9 +75,15 @@ def getenv(key):
     # and we go ahead with the 'desktop' file? Odd.
     FILES = [fn[:-3] + 'c' for fn in FILES if fn.endswith('pyx')]
 
-
-def find_javac(possible_homes):
+    
+    
+possible_homes = [getenv('JDK_HOME'), getenv('JAVA_HOME'), getenv('JRE_HOME'),
+                  getenv('JDK_HOME\bin'), getenv('JAVA_HOME\bin'), getenv('JRE_HOME\bin')]
```

Comment:
Have you actually looked at what `getenv()` really does?
https://github.com/kivy/pyjnius/blob/5e19dbe425a6336e94b2f97c4b66c32ed1cd8652/setup.py#L23

None of these `bin` folders work and no one sets an environment variable with a backslash.